### DISCUSSION TITLE
About | how.wtf

### DISCUSSION THREAD

**Discussion by giscus** [thomasnotfound/how.wtf#58] [Blog Posts]
# About | how.wtf

https://how.wtf/about.html

**Comment by tif-calin** [comment]
What an incredible domain name you snagged

**Reply by thomasgtaylor** [reply]
Thank you! I’m still excited that I was able to snag it.

**Comment by vsnthdev** [comment]
This is such a beautifully designed microblog 🤩

Looking forward to more helpful articles on this blog.

**Reply by thomasgtaylor** [reply]
Thank you for your kind words!
